src/SerialFirmata.h

```diff
@@ -142,6 +146,12 @@ class SerialFirmata: public FirmataFeature
     int serialBytesToRead[SERIAL_READ_ARR_LEN];
     signed char serialIndex;
 
+#if defined(SERIAL_STORE_AND_FORWARD)    
+    unsigned long lastReceive[SERIAL_READ_ARR_LEN];
+    unsigned char maxCharDelay[SERIAL_READ_ARR_LEN];
+    int lastAvailableBytes[SERIAL_READ_ARR_LEN];
```

Comment:
this line needs to be declared outside if the ifdef because it is used outside of an ifdef on line 337 in SerialFirmata.cpp